main-site/components/events-calendar-section/EventsCalendarSection.styles.ts

```diff
@@ -1,205 +1,198 @@
-padding-top: 15em;
-padding-bottom: 12em;
-@media ${max.tabletSm} {
-  padding-top: 0.6em;
-}
+  padding-top: 15em;
+  padding-bottom: 12em;
+  @media ${max.tabletSm} {
+    padding-top: 0.6em;
+  }
 `;
 
 const StyledEventsContainer = styled.div`
-top: 3em;
-position: relative;
+  top: 3em;
+  position: relative;
 `;
 
 const StyledEventsBox = styled.div`
-border-radius: 2em;
-background-color: ${colors.BUTTON_DARK_GREEN};
-width: 95%;
-position: relative;
-flex-direction: column;
-display: flex;
-margin-left: 2.5%;
-top: 4%;
-margin-top: 4%;
-margin-bottom: 2%;
-@media ${max.tabletLg} {
-  margin-top: 5%;
-  margin-bottom: 3%;
-  width: 88%;
-  border-radius: 1.7em;
-  margin-left: 2em;
-}
-@media ${max.tablet} {
-  border-radius: 1.3em;
-}
-@media ${max.tabletSm} {
-  width: 90%;
-  margin-left: 5%;
-}
-@media ${max.mobile} {
-  width: 88%;
-  margin-left: 6%;
-}
+  border-radius: 2em;
+  background-color: ${colors.BUTTON_DARK_GREEN};
+  width: 95%;
+  position: relative;
+  flex-direction: column;
+  display: flex;
+  margin-left: 2.5%;
+  top: 4%;
+  margin-top: 4%;
+  margin-bottom: 2%;
+  @media ${max.tabletLg} {
+    margin-top: 5%;
+    margin-bottom: 3%;
+    width: 88%;
+    border-radius: 1.7em;
+    margin-left: 2em;
+  }
+  @media ${max.tablet} {
+    border-radius: 1.3em;
+  }
+  @media ${max.tabletSm} {
+    width: 90%;
+    margin-left: 5%;
+  }
+  @media ${max.mobile} {
+    width: 88%;
+    margin-left: 6%;
+  }
 `;
 
 const StyledTextContainer = styled.div`
-display: flex;
-margin: 4%;
-@media ${max.tablet} {
-  margin: 1.5em;
-}
-@media ${max.tabletSm} {
-  margin: 6%;
-  padding-left: 1em;
-}
-@media ${max.mobile} {
+  display: flex;
   margin: 4%;
-  padding-left: 0.5em;
-}
+  @media ${max.tablet} {
+    margin: 1.5em;
+  }
+  @media ${max.tabletSm} {
+    margin: 6%;
+    padding-left: 1em;
+  }
+  @media ${max.mobile} {
+    margin: 4%;
+    padding-left: 0.5em;
+  }
 `;
 
 const TextLeftContainer = styled.div`
-width: 80%;
-@media ${max.tabletSm} {
-  width: 93%;
-}
+  width: 80%;
+  @media ${max.tabletSm} {
+    width: 93%;
+  }
 `;
 
-
 const StyledEventsCalendar = styled.div`
-border-radius: 2em;
-width: 50%;
-height: 40em;
-background-color: ${colors.BUTTON_GREEN};
-margin-left: 25%;
-position: relative;
-@media ${max.tabletLg} {
-  height: 42em;
-  width: 70%;
-  margin-left: 15%;
-}  
-@media ${max.tablet} {
-  border-radius: 1.3em;
-  height: 36em;
-  margin-left: 5%;
-  width: 90%;
-}
-@media ${max.tabletSm} {
-  height: 28em;
-  margin-left: 5%;
-}
-@media ${max.mobile} {
-  height: 18em;
-} 
+  border-radius: 2em;
+  width: 50%;
+  height: 40em;
+  background-color: ${colors.BUTTON_GREEN};
+  margin-left: 25%;
+  position: relative;
+  @media ${max.tabletLg} {
+    height: 42em;
+    width: 70%;
+    margin-left: 15%;
+  }
+  @media ${max.tablet} {
+    border-radius: 1.3em;
+    height: 36em;
+    margin-left: 5%;
+    width: 90%;
+  }
+  @media ${max.tabletSm} {
+    height: 28em;
+    margin-left: 5%;
```

Comment:
remove the margin-left: 5% since it's a repeat

Reply:
In EventsCalenderSection???

Reply:
oh wait i see